CHANGELOG.txt

```diff
@@ -1,3 +1,7 @@
+development:
```

Comment:
This is on releases, so it'd be better to mark it 0.10.x just to ease merging (if we don't make any more point releases, that is.)